modules/networking-osp-enabling-vfio-noiommu.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+[id="networking-osp-enabling-vfio-noiommu_{context}"]
+= Enabling the No-IOMMU Operator for the {rh-openstack} VFIO driver
```

Comment:
This isn't really a separate operator.  This sequence "Enables the No-IOMMU feature for the ..."

Same as for below references to an operator.

Reply:
@atyronesmith Sure. Had it in my head that this triggered an operator at some point. 

LMK if https://github.com/openshift/openshift-docs/pull/28315/commits/4633a9da29e072db473a9b0618554e67c2d0cb26 and https://github.com/openshift/openshift-docs/pull/28315/commits/e6eabbe0854d870cfeece9ba07799929ce0efb12 work for you.